test/functional/provider/admin/applications_controller_test.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
```

Comment:
We shouldn't be adding any new tests in `test/functional`. Please favour `test/integration` instead.